src/components/dialog-full-screen/dialog-full-screen.spec.js

```diff
@@ -162,6 +131,80 @@ describe('DialogFullScreen', () => {
   });
 });
 
+describe('closeIcon', () => {
+  let wrapper;
+  let spy;
+
+  beforeEach(() => {
+    jest.restoreAllMocks();
+    spy = jest.fn();
+
+    wrapper = mount(
+      <DialogFullScreen
+        open
+        onCancel={ spy }
+        onConfirm={ () => { } }
+        title='Test'
+        data-role='baz'
+        data-element='bar'
+      />
+    );
+  });
+
+  it('should close Icon with enter key', () => {
+    const closeIcon = wrapper.find(Icon);
+    closeIcon.props().onKeyDown({ which: 13 });
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should not close Icon with other key than enter', () => {
```

Comment:
What about ESC key, shouldn't we handle closing when either ESC and Enter are used?

Reply:
You are right. Probably the test is wrongly described. ESC works in the component very well :)

Reply:
please can you update that description then? Or/And maybe setup another it block testing to test that indeed ESC key closes (same way you have enter key tested on line 154)